modules/distr-tracing-otel-send-traces-and-metrics-to-otel-collector-without-sidecar.adoc

```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
+//
+// * /distr_tracing/distr_tracing_otel/distr-tracing-otel-using.adoc
+
+:_content-type: PROCEDURE
+[id="distr-tracing-otel-send-traces-and-metrics-to-otel-collector-no-sidecar_{context}"]
+= Sending traces and metrics to the OpenTelemetry Collector without sidecar injection
+
+You can set up sending telemetry data to an OpenTelemetryCollector instance without sidecar injection, which involves manually setting several environment variables.
+
+.Prerequisites
+
+* The {TempoName} is installed and a TempoStack instance is deployed.
+
+* You have access to the cluster through the OpenShift web console or the OpenShift CLI (`oc`):
+
+** You are logged in to the OpenShift web console as a cluster administrator with the `cluster-admin` role.
+
+** An active OpenShift CLI (`oc`) session by a cluster administrator with the `cluster-admin` role.
+
+** For {product-dedicated}, you must have an account with the `dedicated-admin` role.
+
+.Procedure
+
+. Create an OpenShift project for the OpenTelemtry Collector.
++
+[source,yaml]
+----
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: observability
+----
+
+. Create a service account.
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: otel-collector-deployment
+  namespace: observability
+----
+
+. Grant permissions to the service account for the `k8sattributes` and `resourcedetection` processors.
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: otel-collector
+rules:
+- apiGroups: ["", "config.openshift.io"]
+  resources: ["pods", "namespaces", "infrastructures", "infrastructures/status"]
+  verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: otel-collector
+subjects:
+- kind: ServiceAccount
+  name: otel-collector-sidecar
```

Comment:
I think we need to change this and remove the sidecar from name to avoid confussion